index.js

```diff
@@ -34,7 +34,7 @@ const checkCwdOption = options => {
 	}
 };
 
-const getPathString = p => p.stats instanceof fs.Stats ? p.path : p;
+const getPathString = fastGlobResult => fastGlobResult.path || fastGlobResult;
```

Comment:
This function mean to handle fastGlobResult, but `p.stats` is not always true, see  [`objectMode` option in `fast-glob`](https://github.com/mrmlnc/fast-glob#objectmode)